irs_lqr/irs_lqr_first_order.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+from irs_lqr.irs_lqr import IrsLqr
+
+class IrsLqrFirstOrder(IrsLqr):
+    def __init__(self, system, sampling,
+        Q, Qd, R, x0, xdt, u_trj_initial, xbound, ubound, solver_name="osqp"):
+        super(IrsLqrFirstOrder, self).__init__(system, Q, Qd, R, x0, xdt,
```

Comment:
We can call `super` without referring to the base class, as mentioned here: https://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods

Reply:
Done. Apparently I never moved on from some habits from 2.7....